docs: add database creation guide to CONTRIBUTION file (#219)
mikro-orm/mikro-orm: CONTRIBUTING.md

- Fork the project, install NPM dependencies and start docker to have all databases ready:

    ```sh
    yarn
    docker-compose up -d
    ```

- You have to create MySQL and PostgreSQL databases manually using the following commands.
    
    ```sh

>     sh tests/scripts/sdatabase_setup.sh

Reviewer: typo (`sdatabase_setup`)

    ```

> - You can skip the following section part if the script works fine for you.

Reviewer: drop the `part` word (`You can skip the following section if the script works fine for you.`)

    ```sh
    docker ps
    ```

    **Read the names of the containers, there should be 3 containers running.**

>     #### MYSQL

Reviewer: Please use `MySQL`

    ```sh
    docker-compose exec <name_of_mysql_container> bash
    mysql
    create database mikro_orm_test;
    exit
    exit
    ```

>     #### POSTGRESQL

Reviewer: Please use `PostgreSQL`

    docker-compose up -d
    ```

> - You have to create the databases manually using the following commands.

Reviewer: ```suggestion
- You have to create MySQL and PostgreSQL databases manually using the following commands.
```

>     ```shell

Reviewer: ```suggestion
    ```sh
```

    sh scripts/database_setup.sh
    ```

> - You can skip this part if the script works fine for you.

Reviewer: ```suggestion
- You can skip following section if the script works fine for you.
```

>     ```shell

Reviewer: ```suggestion
    ```sh
```

    docker ps
    ```

    **Read the names of the containers, there should be 3 containers running.**

>     ## MYSQL

Reviewer: ```suggestion
    #### MySQL
```

    ```shell
    docker-compose exec <name_of_mysql_container> bash
    mysql
    create database mikro_orm_test;
    exit
    exit
    ```

>     ## POSTGRE

Reviewer: ```suggestion
    #### PostgreSQL
```
